server/src/cmds/run/mod.rs

```diff
@@ -43,18 +43,9 @@ pub struct Args {
 // They seem to be correct for Linux and macOS at least.
 const LOCALTIME_PATH: &str = "/etc/localtime";
 const TIMEZONE_PATH: &str = "/etc/timezone";
-const ZONEINFO_PATHS: [&str; 2] = [
-    "/usr/share/zoneinfo/",       // Linux, macOS < High Sierra
-    "/var/db/timezone/zoneinfo/", // macOS High Sierra
-];
-
-fn trim_zoneinfo(path: &str) -> &str {
-    for zp in &ZONEINFO_PATHS {
-        if let Some(p) = path.strip_prefix(zp) {
-            return p;
-        }
-    }
-    path
+
+fn zoneinfo_name(path: &str) -> Option<&str> {
```

Comment:
Could you add a comment here mentioning the common path formats we've seen? e.g. the two that were already listed, and the new nix one.

Reply:
Ok